src/core/email/templates/progress-report-status-changed.template.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Button,
+  Column,
+  HideInText,
+  Section,
+  Text,
+} from '@seedcompany/nestjs-email/templates';
+import { startCase } from 'lodash';
+import { DateTime } from 'luxon';
+import { EmailReportStatusNotification } from '../../../components/progress-report/workflow/progress-report-workflow.service';
+import { fullName } from '../../../components/user/fullName';
+import { EmailTemplate, Heading } from './base';
+import { useFrontendUrl } from './frontend-url';
+
+export function ProgressReportStatusChanged({
+  changedBy,
+  project,
+  languageEngagement,
+  report,
+  reportQtrLabel,
+  newStatusVal,
+  previousStatusVal,
+}: EmailReportStatusNotification) {
+  const projectUrl = project?.id
+    ? useFrontendUrl(`/projects/${project.id}`)
+    : useFrontendUrl(`/projects/no-project-found`);
+  const projectName = project?.name.value;
+
+  const reportUrl = report.id
+    ? useFrontendUrl(`/progress-reports/${report.id}`)
+    : useFrontendUrl(`/projects/no-project-reports-found`);
+  const reportLabel = reportQtrLabel
+    ? `Quarterly Report - ${reportQtrLabel}`
+    : 'Quarterly Report';
+
+  const changerUrl = useFrontendUrl(`/users/${changedBy.id}`);
+  const changerName = fullName(changedBy);
+
+  const oldStatus = startCase(previousStatusVal) || undefined;
+  const newStatus = startCase(newStatusVal) || undefined;
+
+  const changedAt = DateTime.now()
+    .setZone(changedBy.timezone.value ?? 'America/Chicago')
+    .toLocaleString(DateTime.DATETIME_FULL);
+
+  return (
+    <EmailTemplate
+      title={
+        reportLabel && oldStatus && newStatus
+          ? `${reportLabel} changed from ${oldStatus} to ${newStatus}`
+          : `${reportLabel} Status Change`
+      }
+    >
+      <Heading>
+        {reportLabel && newStatus ? (
+          <>
+            {reportLabel} is now <em>{newStatus}</em>
+          </>
+        ) : (
+          `${reportLabel} has a new status`
+        )}
+      </Heading>
+
+      <Section>
+        <Column>
+          <Text paddingBottom={16}>
+            {changerName ? <a href={changerUrl}>{changerName}</a> : 'Someone'}{' '}
+            has changed <a href={reportUrl}>{reportLabel}</a>{' '}
+            {newStatus ? (
+              <>
+                {oldStatus ? (
+                  <>
+                    from <em>{oldStatus}</em>{' '}
+                  </>
+                ) : null}
+                to <em>{newStatus}</em>{' '}
+              </>
+            ) : null}
+            for <a href={projectUrl}>{projectName}</a> - {languageEngagement} on{' '}
```

Comment:
I'd mention the engagement name first as it's the most important and identifiable piece of information we have.
User's will be spammed with these notifications when the end of the quarter rolls around and they will all be for the same quarter.

In fact I would put that label before the quarter label in the subject as well.

Otherwise the email box is
```
Q1 FY2023 changed from Not Started to In Progress
Q1 FY2023 changed from Not Started to In Progress
Q1 FY2023 changed from Not Started to In Progress
Q1 FY2023 changed from Not Started to In Progress
Q1 FY2023 changed from Not Started to In Progress
Q1 FY2023 changed from Not Started to In Progress
```
GIven that we really only have one project for a language going at once too, I'd drop the project name in the subject for terseness.
```
{language} {quarter} now {newStatus}
```
```
Mankon Q1 FY2023 now In Progress
```

Though maybe the project name would help. Might be worth trying/asking both and see which one is better.
```
{language} ({project}) {quarter} now {newStatus}
```
```
Mankon (Abakwa Oral Cluster) Q1 FY2023 now In Progress
```